examples/checkpointing/checkpoint.py

```diff
@@ -13,7 +13,7 @@ class CheckpointOperator(Operator):
                     automatically include these cached arguments in the call to the
                     underlying devito.Operator.apply().
     """
-    t_arg_names = {'t_start': 't_s', 't_end': 't_e'}
+    t_arg_names = {'t_start': 'time_s', 't_end': 'time_e'}
```

Comment:
this hardcoding is bad, although it was this way in the previous PR too...

Reply:
Containing this hardcoding is the reason this wrapper exists.